recipes/hpx/bld.bat

```diff
@@ -0,0 +1,23 @@
+mkdir build
+cd build
+if %errorlevel% neq 0 exit /b %errorlevel%
+
+cmake %SRC_DIR% ^
+    -G "Visual Studio 16 2019" -A x64 ^
+    -D CMAKE_BUILD_TYPE="Release" ^
+    -D CMAKE_INSTALL_PREFIX:PATH="%PREFIX%" ^
```

Comment:
```suggestion
    -D CMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
```
Maybe you won't have to specify HWLOC below too?

Reply:
Thank you for your suggestions. I have updated the build scripts and am waiting for the builds to pass. In another project the HWLOC stuff was necessary, but I hope it is not anymore, given that we now use `LIBRARY_PREFIX`.